src/config/rc-files.md

```diff
@@ -1,4 +1,4 @@
-# rc.conf, rc.local and rc.shutdown
+# rc Files
```

Comment:
I think I prefer the original name. What do you think?

Reply:
i don't mind it, it's just that it's rather long, which is not ideal for the ToC on the Web site. Happy to change it if you think this isn't an issue.

Reply:
Just tested it locally, it has the same width as the title for "Session management", so it looks like it should be fine.